kube-burner-workload/env

```diff
@@ -21,8 +21,9 @@ KUBECONFIG=
 
 # PLATFORM is one of the folders under network-policy workload
 PLATFORM=openshift
-# CONVERGENCE_TRACKER
+# Convergence tracker settings
 CONVERGENCE_TRACKER=true
+CONVERGENCE_TIMEOUT=3600
```

Comment:
I think this one should be next to THRESHOLD, which is also a convergence tracker parameter. And we probably want to explain the difference between them (and probably rename THRESHOLD to something better), and explain that CONVERGENCE_TIMEOUT includes THRESHOLD (so should always have greater value)

Reply:
I change `threshold` to `convergence_period`. I think it should ideally be called `convergence_stability_period`, but maybe it's too long, so I chose `convergence_period`. WDYT?